README.md

```diff
@@ -1,15 +1,87 @@
 # Gorilla REPL
+This is a modified version of Gorilla, a rich REPL for Clojure in the notebook style. If you're interested in the original version, you should take a look at its [website](http://gorilla-repl.org). This version adds the ability to open any Clojure file in Gorilla REPL, as well as to save a worksheet
+without Gorilla markup to allow for easy collaboration.
 
-Gorilla is a rich REPL for Clojure in the notebook style. If you're interested you should take a look at its
-[website](http://gorilla-repl.org).
+Secrets of Gorilla REPL
```

Comment:
I think it would be better to have "Secrets of Gorilla REPL" as  separate document, possibly on the website rather than in the repo. We could then revert the README to the original, with the addition maybe of a link to the "secrets" doc.